queries/appSettings.ts

```diff
@@ -102,24 +95,24 @@ export const getUploadthingVariables = createCachedFunction(async () => {
     UPLOADTHING_SECRET,
     UPLOADTHING_APP_ID,
```

Comment:
I don't think the v7 API actually uses the App ID (could be wrong).

Also, we don't have to keep the capitalisation from the env variables. Might be better to use pascalCase for consistency.